src/components/Chat/Chat.js

```diff
@@ -149,8 +135,11 @@ function Chat(props) {
             });
 
             const responseJson = await fetchCreateVoiceRecording(file, JSON.stringify(transcriptData), transcript);
-            _voiceRecordings.push(responseJson.data);
-            _setVoiceRecordings([..._voiceRecordings]);
+
+            await props.dispatch({
```

Comment:
This was saving state on the component level. It is expected that this stores state on a global level via Redux. This fixes that 👍